web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -276,10 +279,14 @@ export default class MainProcess {
       }
     });
 
+    ipcMain.handle('main-process-open-config-file', () => {
+      shell.openPath(this.configFileStorage.getFilePath());
```

Comment:
`shell.openPath` is cool when it works and pretty bad when it doesn't. My Mac doesn't have a default program configured for JSON files so the button opened Xcode. But as I don't have Xcode installed it just shows the installer for it and that's it.

What do you think about making this IPC handler return the path to the config file so that after opening the file we can show an info notification? Something like "Opened the config file at (full path here) in the default editor".

Reply:
I supposed that the file would be opened in TextEdit in that case :/ Thanks for pointing this.

Your workaround seems to be fine. Showing a notification is unnecessary for situations where it "works", but we can't do anything better :(

Reply:
> Showing a notification is unnecessary for situations where it "works", but we can't do anything better :(

When it works the user might not notice the notification at all (because they'll be in the editor) and in any case it'll disappear after a few seconds by itself.